<launch>

  <node pkg="rosbag" type="record" name="turtlebot_record"
        args="record -O $(find homework2)/rosbag/rosbag_record /scan /odom /amcl_pose cmd_vel_mux/input/teleop"/>  

</launch>
